plugins/gatsby-source-cd-airtable/src/load-data.ts

```diff
@@ -0,0 +1,27 @@
+import fetch from 'node-fetch'
+
+export const loadData = <T>(tableName: string): Promise<T[]> => {
+  if (!process.env.AIRTABLE_API_KEY || !process.env.AIRTABLE_BASE_KEY) {
+    throw new Error('API key and base key are both required')
+  }
+  if (!tableName) {
+    throw new Error('Table Name is required')
+  }
+  return fetch(
+    `https://api.airtable.com/v0/${process.env.AIRTABLE_BASE_KEY}/${tableName}`,
+    {
+      headers: {
+        Authorization: `Bearer ${process.env.AIRTABLE_API_KEY}`,
+      },
+    }
+  )
+    .then((data) => {
```

Comment:
Je nějaký důvod pro použití promises místo async/await? Hlasoval bych pro to druhé :-)

Reply:
Já mám většinou radši promises, ale klidně to upravím. Vím že jsem jeden z mála :))